addons/website/static/src/builder/plugins/options/background_option.js

```diff
@@ -21,9 +21,11 @@ export class WebsiteBackgroundOption extends BaseOptionComponent {
         super.setup();
         const { showColorFilter } = useBackgroundOption(this.isActiveItem);
         this.showColorFilter = () => showColorFilter() || this.isActiveItem("toggle_bg_video_id");
+        const parallaxBgSelector =
+            ":scope > .s_parallax_bg, :scope > .s_parallax_bg_wrap > .s_parallax_bg";
         this.websiteBgOptionDomState = useDomState((el) => ({
             // Only search for .s_parallax_bg that are direct children
-            applyTo: el.querySelector(":scope > .s_parallax_bg") ? ".s_parallax_bg" : "",
+            applyTo: el.querySelector(parallaxBgSelector) ? ".s_parallax_bg" : "",
```

Comment:
Is not this a bug to have "applyTo" which does not look for direct children? Should not it be fixed first?

Reply:
> Is not this a bug to have "applyTo" which does not look for direct children? Should not it be fixed first?

`:scope >` is only used for the condition

Reply:
That's what I am saying yes... is not this a bug to only use it for the condition? :disappointed:

Reply:
> Is not this a bug to have "applyTo" which does not look for direct children? Should not it be fixed first?

No, this is not a bug. In fact, even if all `.s_parallax_bg` elements match the selector, only the first one is retrieved later because the option uses `getEditingElement` and not `getEditingElements` (with an "s"), see where `applyTo` is handled in the `useBuilderComponent function`. So it’s not a bug, but I’ll still change it to avoid any confusion 😉